compiler/modules/front/codeGen.bal

```diff
@@ -160,21 +160,21 @@ class CodeGenContext {
         self.loopContext = (<LoopContext>self.loopContext).enclosing;
     }
 
-    function onBreakLabel() returns bir:Label|err:Semantic {
+    function onBreakLabel(bir:Position pos) returns bir:Label|err:Semantic {
```

Comment:
We are using a mix of d:Position, bir:Position and s:Position.  I suggest we do this once:

```
type Position d:Position;
```

and then use simply `Position` throughout this module.

Reply:
fixed